### ISSUE TITLE
Cursor 0.33.1 doesn't store my OpenAI API key any more?

### ISSUE THREAD

**Issue by chrisk477** [getcursor/cursor#1440]
Currently I get an error each time I restart Cursor (even though I generated a new "project" OpenAI API key).

Soooo, is this a way to tell me to either subsribe to Cursor Pro or to leave?

**Comment by truell20** [comment]
API keys are definitely supported and if there's a problem we'd certainly want to fix. 

I'm having a hard time reproducing on my end. To help us figure out what's going on, can you verify that your API key is properly set in settings and that making some basic cURL request with it outside of Cursor works?

**Comment by truell20** [comment]
Also, are you doing anything special like overriding the openai baseurl?

**Comment by chrisk477** [comment]
Sorry if I sounded snarky in my original message.
In the last few days, I generated multiply API keys, pasted them in the settings dialog, successfully verified them and then got an error message after the next restart of Cursor.
Now, when I tried it a few minutes ago, it worked fine after a restart and even after a sytem restart. Maybe it's just something with my system or a server timeout or something like that.
